### PR TITLE
Links - Multistage Abused Services

### PR DESCRIPTION
**Pull request by zoomequipd**
# Description

Detection rules for two abused services (Docusign and Adobe Express) to host documents with links to phishing pages. 

# Associated samples

## Docusign
- [Sample 1](https://platform.sublimesecurity.com/messages/1f708d176b12c733de110057a367f2902df0e84f3bc1d4caded0b2ba94902b49)
## Adobe Express
- [Sample 1](https://platform.sublimesecurity.com/messages/ed30032c676e373c349cce5f2a41c1fe63e066295bfe73abb011566abe8b4f32)

### CODE REVIEW

**Comment by zoomequipd** [comment]
marking as ready as it just flagged on a abused docusign campaign

File: detection-rules/link_multistage_docusign.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  // reply-to email address as never been sent an email by the org
+  and not any(headers.reply_to, .email.email in $recipient_emails)
+  
+  // message is from docusign actual
+  and sender.email.domain.domain == 'docusign.net'
+  and headers.auth_summary.spf.pass
+  and headers.auth_summary.spf.details.designator == 'docusign.net'
+  and headers.auth_summary.dmarc.pass
+  
+  // filter out all the links, keeping just the one I want
```
Comment (morriscode) [inline detection-rules/link_multistage_docusign.yml:17]:
```suggestion
```

File: detection-rules/link_multistage_docusign.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  and any(filter(body.links,
+                 // find the link I want to test, this should be a docusign link
```
Comment (morriscode) [inline detection-rules/link_multistage_docusign.yml:19]:
```suggestion
                 // Target the DocuSign link
```

Comment (morriscode) [reply detection-rules/link_multistage_docusign.yml:19]: Or accept the previous suggestion and delete this comment?

**Review by morriscode — APPROVED** [review]

**Pull request merged**